iotilecore/iotile/core/hw/reports/utc_assigner.py

```diff
@@ -23,7 +23,7 @@
 import datetime
 import logging
 from bisect import bisect_left
-from typedargs.exceptions import ArgumentError
+from typedargs.exceptions import ArgumentError, KeyValueException
```

Comment:
extra `KeyValueException` import is no longer needed.

Reply:
removed